ibm-mq/1.6/modules/ROOT/pages/ibm-mq-xml-ref.adoc

```diff
@@ -549,7 +549,11 @@ Allows a user to perform a session recover when the AckMode#MANUAL mode is elect
 
 ** CORRELATION_ID
 ** MESSAGE_ID
-** NONE |  |  +++CORRELATION_ID+++ |
+** NONE | Indicates how requests and reply messages are correlated, which ensures that the incoming message is the proper reply to the outgoing message.
+
+* The `CORRELATION_ID` pattern value implies that the *Publish Consume* operation, after publishing a message to the destination queue, consumes a message using a selector that expects a message with the same correlation ID used to publish the request message.
+
+* The `MESSAGE_ID` pattern value implies that the *Publish Consume* operation, after publishing a message to the destination queue, consumes a message using a selector expecting a message with a correlation ID of the same value as the message ID of the request message. | CORRELATION_ID |
```

Comment:
Ditto the above with slight variations:

The `MESSAGE_ID` pattern value is used with the *Publish Consume* operation. After the operation publishes a request message to the destination queue, it expects a message with the same correlation ID as the request message. It then consumes that correlated response message.